gridstatus/aeso/aeso.py

```diff
@@ -598,3 +598,170 @@ def get_unit_status(
                 "Dispatched Contingency Reserve",
             ]
         ]
+
+    @support_date_range(frequency="31D")
+    def get_generator_outages_hourly(
+        self,
+        date: str | pd.Timestamp | tuple[pd.Timestamp, pd.Timestamp],
+        end: str | pd.Timestamp | tuple[pd.Timestamp, pd.Timestamp] | None = None,
+        verbose: bool = False,
+    ) -> pd.DataFrame:
+        """
+        Get hourly generator outage data.
+
+        Args:
+            date: Start date for the data. Can be "latest" to get current data.
+            end: End date for the data. If not provided, will get 24 months of data.
+            verbose: Whether to print verbose output.
+
+        Returns:
+            DataFrame containing generator outage data
+        """
+        if date == "latest":
+            current_time = pd.Timestamp.now()
+            return self.get_generator_outages_hourly(
+                date=current_time,
+                end=current_time + pd.DateOffset(months=4),
+            )
+        else:
+            start_date = pd.Timestamp(date)
+            if start_date.tz is None:
+                start_date = start_date.tz_localize(self.default_timezone)
+            else:
+                start_date = start_date.tz_convert(self.default_timezone)
+
+            end_date = pd.Timestamp(end) if end else None
+            if end_date is not None:
+                if end_date.tz is None:
+                    end_date = end_date.tz_localize(self.default_timezone)
+                else:
+                    end_date = end_date.tz_convert(self.default_timezone)
+
+        endpoint = f"aiesgencapacity-api/v1/AIESGenCapacity?startDate={start_date.strftime('%Y-%m-%d')}"
+        if end_date:
+            endpoint += f"&endDate={end_date.strftime('%Y-%m-%d')}"
+        data = self._make_request(endpoint)
+
+        fuel_type_mapping = {
+            "GAS": "Gas",
+            "COAL": "Coal",
+            "HYDRO": "Hydro",
+            "WIND": "Wind",
+            "SOLAR": "Solar",
+            "ENERGY STORAGE": "Energy Storage",
+            "OTHER": "Biomass and Other",
+        }
+
+        sub_fuel_type_mapping = {
+            "SIMPLE_CYCLE": "Simple Cycle",
+            "COMBINED_CYCLE": "Combined Cycle",
+            "COGENERATION": "Cogeneration",
+            "GAS_FIRED_STEAM": "Gas Fired Steam",
```

Comment:
For `AESO().get_fuel_mix()`, we have columns for all of these gas subtypes. I saw you talk about combining these in the description. Given the existing fuel mix and the fact that when we sum these to gas, we are losing information, I'd lean in the direction of making these all separate columns.